test/cloudbuild/mkp_verify.yaml

```diff
@@ -0,0 +1,21 @@
+steps:
+- id:   'pullDeployer'
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['pull', 'gcr.io/$PROJECT_ID/hosted/$COMMIT_SHA:${_MM_VERSION}']  
+- id: "verify"
+  waitFor: ['pullDeployer']
```

Comment:
When this is a simple sequential job, there's no need to write `waitFor`

Reply:
I remember hit the issue saying no such image and it didn't pull image.
you may correct me. 

prefer keep it.

By the way, do you know how to test such post-submit before a submit? via calling script directlly? (setup ADC etc. first)

Reply:
I put the notes to comments on how to test the changes before submit